st2api/st2api/controllers/v1/actionalias.py

```diff
@@ -58,6 +74,39 @@ def get_all(self, **kwargs):
     def get_one(self, ref_or_id):
         return super(ActionAliasController, self)._get_one(ref_or_id)
 
+    @jsexpose(arg_types=[str], body_cls=ActionAliasMatchAPI, status_code=http_client.ACCEPTED)
+    def match(self, action_alias_match_api, **kwargs):
```

Comment:
It's neither validation nor testing though, because you would validate or test against something. "Match" is probably both the closest and the most clear word for what's happening. :)